store/router.go

```diff
@@ -189,7 +189,7 @@ func (r *Router) multiSourceRead(ctx context.Context, commitment []byte, fallbac
 			continue
 		}
 		// verify cert:data using EigenDA verification checks
-		err = r.eigenda.Verify(key, data)
+		err = r.eigenda.Verify(commitment, data)
```

Comment:
Is it true that the only cached source is eigenda?

Reply:
I am not super familiar with multisource

Reply:
nah the cached sourced can be arbitrary,  but we'll need to do the eigenda verifications against it to ensure the data hasn't been tampered with